controllers/admin/AdminImportController.php

```diff
@@ -1635,6 +1635,16 @@ protected function productImportOne($info, $default_language_id, $id_lang, $forc
 
         $product = new Product($id_product);
 
+        if (!$product->id && empty($info['name'])) {
+            $this->errors[] = sprintf(
+                $this->trans('Product with reference %1$s (ID: %2$s) cannot be saved: product name is missing', [], 'Admin.Advparameters.Notification'),
```

Comment:
```suggestion
                $this->trans('Product with reference %1$s (ID: %2$s) could not be saved because the product name is missing.', [], 'Admin.Advparameters.Notification'),
```